# AAV-amplicon_QCBio
Scripts and files for computing the enrichment of AAV amplicon (project personnel involved: Prof. Melissa Spencer and Dr. Michael Enami at UCLA)


For each sample, we keep the sequenced reads from the demultiplexed Qseq files where the primer sequences are present and precisely matched. The sequencing was performed as pair-ended, and we keep only the target sequences if the forward and backward reads are consistent (script: find_r1r2consensus.R). Variants of 21 bps length enclosed by the primer sequences are further extracted (script: find_target.py), and are translated into peptide sequences of 7 amino acids length (script: translate_target.py). We compute the abundance of variants by counting the number of all unique variants in a given sample, and we further normalize the abundance by the sequencing reads of the sample (script: enrichment_analysis_all.R; file: read_depth.txt). The enrichment score (E) of each variant is computed as the ratio between the normalized abundance in P2 sample versus P1 sample (sample output file: comp_xxx_sort.csv; sample plot: comparison_xxx.pdf). 
